<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Atacar</class>
 <widget class="QDialog" name="Atacar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/fondos/fondomov.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mov">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/movimientos.png);
background: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="a1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pp1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>104</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="inf1">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>107</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/tipos/NONE.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mov_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/movimientos.png);
background: transparent;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mov_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>170</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/movimientos.png);
background: transparent;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mov_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/movimientos.png);
background: transparent;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="a2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="a3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="a4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pp2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>105</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>88</string>
   </property>
  </widget>
  <widget class="QLabel" name="pp3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>274</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>88</string>
   </property>
  </widget>
  <widget class="QLabel" name="pp4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>274</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>88</string>
   </property>
  </widget>
  <widget class="QLabel" name="inf2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>107</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/tipos/NONE.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inf3">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>276</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/tipos/NONE.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inf4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>276</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/tipos/NONE.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pp11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>104</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>88</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pp22">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>105</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>88</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pp33">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>274</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>88</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pp44">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>274</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>88</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="attck2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="attck3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="attck1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
</string>
   </property>
   <property name="text">
    <string>Giro Infernal Fuego</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="attck4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>fondo</zorder>
  <zorder>mov</zorder>
  <zorder>pp1</zorder>
  <zorder>inf1</zorder>
  <zorder>mov_2</zorder>
  <zorder>mov_3</zorder>
  <zorder>mov_4</zorder>
  <zorder>pp2</zorder>
  <zorder>pp3</zorder>
  <zorder>pp4</zorder>
  <zorder>inf2</zorder>
  <zorder>inf3</zorder>
  <zorder>inf4</zorder>
  <zorder>pp11</zorder>
  <zorder>pp22</zorder>
  <zorder>pp33</zorder>
  <zorder>pp44</zorder>
  <zorder>attck2</zorder>
  <zorder>attck3</zorder>
  <zorder>attck1</zorder>
  <zorder>attck4</zorder>
  <zorder>a4</zorder>
  <zorder>a1</zorder>
  <zorder>a2</zorder>
  <zorder>a3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
